### PR TITLE
[WebProfilerBundle] Display uploaded files in the profiler

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #26926
| License       | MIT
| Doc PR        | -

profiler_files

Note: I wanted to use the VarDumper to display the table information (as we do in the rest of tables) but I had lots of problems and I just wanted to create a proof of concept for the feature to see if we like it.

### CODE REVIEW

**Review by phansys — APPROVED** [review]

**Comment by chapterjason** [comment]
Looks awesome, as I watched in the `UploadedFile` I saw the `
getError()` and `getErrorMessage()` what happens if there is just an error with one file?

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig [hunk 1]
```diff
@@ -131,6 +131,35 @@
...
+                        <p>No files were uploaded</p>
+                    </div>
+                {% else %}
+                    <table>
+                        <thead>
+                            <tr>
+                                <th scope="col">File Name</th>
+                                <th scope="col">MIME Type</th>
+                                <th scope="col text-right">Size (bytes)</th>
+                                <th scope="col">Temporary Location</th>
```
[the remark this answers is not in the document]
Comment (chapterjason) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig]: You're right.

```
The file will be deleted from the temporary directory at the end of the request if it has not been moved away or renamed.
```
http://php.net/manual/en/features.file-upload.post-method.php

[the remark this answers is not in the document]
Comment (fabpot) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig]: @javiereguiluz Can you remove this column?

**Review by fabpot — CHANGES_REQUESTED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @javiereguiluz.

**Comment by xabbuh** [comment]
Wouldn't it make sense to also display the error (possibly human readable) if any occurred during uploading?

**Pull request merged**